apps/tutorial/tests/filter_test.py

```diff
@@ -0,0 +1,181 @@
+import random
+import typing
+
+from apps.project.factories import OrganizationFactory, ProjectFactory
+from apps.project.models import (
+    ProjectTypeEnum,
+)
+from apps.tutorial.factories import (
+    TutorialFactory,
+)
+from apps.tutorial.models import (
+    TutorialStatusEnum,
+)
+from apps.user.factories import UserFactory
+from main.tests import TestCase
+
+
+class TestTutorialFilterQuery(TestCase):
+    class Query:
+        TUTORIALS_WITH_FILTERS = """
+            query Tutorials($pagination: OffsetPaginationInput, $filters: TutorialFilter) {
+              tutorials(pagination: $pagination, filters: $filters) {
+                totalCount
+                pageInfo {
+                  offset
+                  limit
+                }
+                results {
+                  id
+                  clientId
+                  name
+                  status
+                  projectId
+                }
+              }
+            }
+        """
+
+    @typing.override
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.user = UserFactory.create()
+        cls.user_resource_kwargs = dict(
+            created_by=cls.user,
+            modified_by=cls.user,
+        )
+
+        cls.organization = OrganizationFactory.create(**cls.user_resource_kwargs)
+
+        cls.project1, cls.project2, cls.project3 = ProjectFactory.create_batch(
+            3,
+            **cls.user_resource_kwargs,
+            project_type=random.choice(  # noqa:S311
+                [
+                    ProjectTypeEnum.FIND,
+                    ProjectTypeEnum.COMPARE,
+                    ProjectTypeEnum.VALIDATE,
+                    ProjectTypeEnum.COMPLETENESS,
+                ],
+            ),
```

Comment:
@thenav56 Should we use random here? Or, just hard-code some project type?

Reply:
We should also check filter by project type because that would be the most useful.